utilities/roswtf/scripts/getlogball.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
It'd be great to drop the `sh` suffix so that there's the option to implement this in Python later (or now).